ilastik/applets/pixelClassification/opPixelClassification.py

```diff
@@ -563,6 +564,24 @@ def __init__(self, *args, **kwargs):
         self.UncertaintyEstimate.connect( self.opUncertaintyCache.Output )
 
     def setupOutputs(self):
+        input_dtype = self.InputImage.meta.dtype
+
+        # When from other libraries, the dtype could also be
+        # numpy.dtype('uint8'), which would not be the same as numpy.uint8
+        assert not isinstance(input_dtype, np.dtype)
+
+        if  np.dtype(input_dtype).kind in np.typecodes['AllInteger']:
```

Comment:
```suggestion
        if  np.dtype(input_dtype).char in np.typecodes['AllInteger']:
```

Reply:
i confirm that this fixes at least the general idea of actually using the predictions of the first stage in the second one.

Reply:
Unfortunately this does not fix the tests, but we are getting there =)

Reply:
I am really sorry for not catching this, I saw the `dtype.kind` check on stackoverflow. I checking my testing scratches notebook, I  was testing this then with `np.int8` for which it actually would work :/